File: func_cube.java
import java.util.Scanner;

public class func_cube {
    public static int cube(int a) {
        int result;
        result = a * a * a;
        return result;
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the number to find out its cubic value:");
        int a = in.nextInt();
        
        System.out.println("Cubic value: " + cube(a));
        
        in.close(); // Close scanner to prevent resource leak
    }
}

File: func_cube.cpp
#include <iostream>
using namespace std;
int cube(int a){
    int result = a*a*a;
    return result;
    
}
int main() {
    int n;
    cout<<"enter the number to find out its cubic value = ";
    cin>>n;
cout<<cube(n);

    return 0;
}
